ibmcloud/image/build.sh

```diff
@@ -34,6 +34,19 @@ if [[ -z "${base_img_path-}" || -z "${dst_img_path-}" || -z "${files_dir-}" ]];
     exit 1
 fi
 
+if [[ -z "${SE_BOOT-}" ]]; then
+    SE_BOOT=0
+fi
```

Comment:
You can write this in a line as follows.

```
SE_BOOT=${SE_BOOT:-0}
```

Reply:
updated.